cpu/mips_pic32_common/periph/gpio.c

```diff
@@ -202,8 +202,12 @@ int gpio_init(gpio_t pin, gpio_mode_t mode)
             break;
         case GPIO_OD_PU:
             pu = 1;
+            output = 1;
+            break;
```

Comment:
Now `od` is not set.
I think the fall-through was intentional, you can make it explicit by adding 
`*/ fall-through */` - gcc will then drop then warning.

Reply:
Should be fixed now.

Reply:
Please squash